pyxem/signals/indexation_results.py

```diff
@@ -148,11 +202,17 @@ def to_crystal_map(self):
         y = xy[0].flatten()
```

Comment:
These arrays don't use the step size. All crystal maps generated with this method will have step size of 1 and a scan unit of "px". That is the intention, correct?

Reply:
For the time being, yes